app/controllers/recommendation_controller.rb

```diff
@@ -1,8 +1,8 @@
 class RecommendationController < ApplicationController
   def index
-    @category = session[:category]
-    @drilldown = session[:drilldown]
-    @questions = session[:questions]
+    @category = Category.last(4).pluck(:name)
```

Comment:
So, this isn't gonna work for us in the long term. Eventually there will be two people who perform the quiz at the same time, someone will see the wrong data.

Let's update the backlog with a ticket to connect the cateogry, drilldown and question choices to the users email